pandas/core/groupby/groupby.py

```diff
@@ -2301,6 +2302,69 @@ def tail(self, n=5):
         mask = self._cumcount_array(ascending=False) < n
         return self._selected_obj[mask]
 
+    def _reindex_output(self, result):
```

Comment:
Did you move this for a particular reason?

Reply:
Nevermind I see the reasoning here since this wasn't available to SeriesGroupBy before